generated/nisync/nisync.proto

```diff
@@ -25,6 +25,7 @@ service NiSync {
   rpc DisconnectSWTrigFromTerminal(DisconnectSWTrigFromTerminalRequest) returns (DisconnectSWTrigFromTerminalResponse);
```

Comment:
I'm guessing my changes will conflict with Zach's recent PR to add some additional get/set methods.  I'll wait for his to go in and resolve conflicts.